### PR TITLE
Moved non-file I/O spec conversion functions into sdk-codegen

### PR DESCRIPTION
**Pull request by jkaster**
- Minor cleanup for testUtils.ts
- Spec file I/O is still in sdk-codegen-scripts
- Fixed some miscellaneous test issues
- Avoids node infection for pending browser-based spec conversion

### CODE REVIEW

File: packages/sdk-codegen-scripts/src/fetchSpec.ts [hunk 1]
```diff
@@ -186,27 +186,16 @@ export const login = async (props: ISDKConfigProps) => {
...
+  const accessToken = await response.access_token
 
-    if (accessToken) {
-      return accessToken
-    } else {
-      log(`Server Response: ${JSON.stringify(response)}`)
-      throw new Error('Access token could not be retrieved.')
-    }
-  } catch (err) {
-    throw err
```
Comment (joeldodge79) [inline packages/sdk-codegen-scripts/src/fetchSpec.ts:209]: 😸

File: packages/sdk-codegen/src/specConverter.ts [hunk 2]
```diff
@@ -0,0 +1,194 @@
+/*
```
Comment (joeldodge79) [inline packages/sdk-codegen/src/specConverter.ts:1]: I'm assuming there's no major re-write in here, just ported from sdk-codegen-scripts?

File: packages/sdk-codegen/src/testUtils/testUtils.ts [hunk 3]
```diff
@@ -59,17 +60,20 @@ export interface ITestConfig {
   testSection: IKeyAny
 }
 
+const homeToRoost = '../../../../'
+
+export const getRootPath = () => path.join(__dirname, homeToRoost)
```
Comment (joeldodge79) [inline packages/sdk-codegen/src/testUtils/testUtils.ts:65]: TIL `__{dir,file}name` - python version is `__file__` - I find the choice of leading and trailing double underscores between languages ... entertaining

**Review by joeldodge79 — APPROVED** [review]
LGTM

**Review by josephaxisa — APPROVED** [review]
👍

File: packages/sdk-codegen-scripts/src/fetchSpec.ts (hunk 1 above)
Comment (jkaster) [reply packages/sdk-codegen-scripts/src/fetchSpec.ts:209]: yeah, I think that was left over from some testing I was doing when first writing that code.

File: packages/sdk-codegen/src/specConverter.ts (hunk 2 above)
Comment (jkaster) [reply packages/sdk-codegen/src/specConverter.ts:1]: Right. This is prep work for full Swagger -> OpenApi on-demand conversion in the browser

**Pull request merged**